### ISSUE TITLE
accessing the application without internet

### ISSUE THREAD

**Issue by jammvis-dro** [talosh/flameTimewarpML#89]
Hey Team, 

Wanted to see if this app can function without internet connectivity or what network info its calling to open up.

**Comment by talosh** [comment]
Hi, v0.4.3 works completely without any connection and v 0.4.5 hopefully
will if I'm able to find out how to host 4gb file. The only need for
internet connection is installing pytorch and othe deps, and pytorch in its
current version goes up to about 4gb for CUDA so it is not possible to
upload release on github.

**Comment by alatteri** [comment]
I'm sure Andy or Randy from Logik forum could figure host these large releases.

> if I'm able to find out how to host 4gb file

**Comment by talosh** [comment]
Yeah that was my first though as well, need some time to get it to some
prod level though. Btw I just had first real life prod case when the shot
has been pulled from "undoable" to "2hrs paint" by setting to finetune over
the weekend. Let's see if more is possible.
